src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -114,20 +117,22 @@ export function CollapsibleNav({
   homeHref,
   storage = window.localStorage,
   onIsLockedUpdate,
+  exitWorkspace,
+  getWorkspaceUrl,
   closeNav,
   navigateToApp,
   navigateToUrl,
   branding,
   ...observables
 }: Props) {
   const navLinks = useObservable(observables.navLinks$, []).filter((link) => !link.hidden);
-  const recentlyAccessed = useObservable(observables.recentlyAccessed$, []);
-  const customNavLink = useObservable(observables.customNavLink$, undefined);
```

Comment:
Do we need to remove `customNavLink$` and `recentlyAccessed$` in the props?

Reply:
These two props are needed for unit tests.

Reply:
Could we delete these in the unit tests file?

Reply:
These two props have been deleted.

Reply:
I recover customNavLink$ and recentlyAccessed$. @SuZhou-Joe suggests that we can keep these two props since we may need them in the future.